rebuild/src/addon.h

```diff
@@ -0,0 +1,132 @@
+#ifndef BLST_TS_ADDON_H__
+#define BLST_TS_ADDON_H__
+
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <string_view>
+#include <memory>
+#include <openssl/rand.h>
+#include "napi.h"
+#include "blst.hpp"
+
+// TODO: these should come out post PR review
+using std::cout;
+using std::endl;
+
+#define BLST_TS_SECRET_KEY_LENGTH 32U
+#define BLST_TS_PUBLIC_KEY_LENGTH_COMPRESSED 48U
+#define BLST_TS_PUBLIC_KEY_LENGTH_UNCOMPRESSED 96U
+#define BLST_TS_SIGNATURE_LENGTH_COMPRESSED 96U
+#define BLST_TS_SIGNATURE_LENGTH_UNCOMPRESSED 192U
+#define BLST_TS_RANDOM_BYTES_LENGTH 8U
+
+#define BLST_TS_SECRET_KEY_LOWER_TAG 0ULL
```

Comment:
what is lower/upper ?

Reply:
That is part of the TypeTag spec

https://nodejs.org/api/n-api.html#napi_type_tag